static/css/base.scss

```diff
@@ -404,3 +404,8 @@ $headings-font-family: "Google Sans", $font-family-sans-serif;
   border-color: var(--dark) !important;
   box-shadow: 0 0 0 0.2rem rgb(0 0 0 / 15%) !important;
 }
+
+/** Form Components */
```

Comment:
I'm not sure what the best practice is for styled components, but I feel like for a component, we should have either have all its styling in a css file or all the styling in its .tsx file, not split over the two because that might make it hard to track down styles?